client/components/support-info/index.jsx

```diff
@@ -17,12 +18,23 @@ function makePrivacyLink( privacyLink, link ) {
 	return null;
 }
 
+/**
+ * Renders a support information component.
+ * @param {Object} props - The component props.
+ * @param {React.ReactNode} [props.children] - The child elements to be rendered inside the component.
+ * @param {string} [props.text] - The text to be displayed in the component.
+ * @param {string} [props.link] - The link URL to be displayed in the "Learn more" section.
+ * @param {string} [props.position] - The position of the popover (default: 'left').
+ * @param {boolean|string} [props.privacyLink] -Boolean whether to display the privacy link. Or the link itself.
```

Comment:
Missing space:

```suggestion
 * @param {boolean|string} [props.privacyLink] - Boolean whether to display the privacy link. Or the link itself.
```

Or do we even need those dashes? I think we can remove them all.